src/electron/menu/dev_menu_template.ts

```diff
@@ -4,10 +4,6 @@ export const devMenuTemplate = (): MenuItem | MenuItemConstructorOptions => {
   return {
     label: 'Development',
     submenu: [
```

Comment:
Why did you remove it from the menu?

Reply:
It does the same as the reset from the main menu, just without confirmation dialog. Do you think it can be useful?

Reply:
That's ok.